packages/paste-core/components/form/src/shared/restricted-attributes.ts

```diff
@@ -0,0 +1,16 @@
+export interface RestrictedPropsTypes {
+  className: undefined;
+  size: undefined;
+  style: undefined;
+  height: undefined;
+  width: undefined;
+  selected?: undefined;
+}
+
+export const restrictedProps: RestrictedPropsTypes = {
```

Comment:
@SiTaggart @richbachman added this here to reduce redundancy.

Reply:
Nice 👍